linkerd/config/src/main/scala/io/buoyant/linkerd/config/types/DtabDeserializer.scala

```diff
@@ -11,3 +11,11 @@ class DtabDeserializer extends ConfigDeserializer[Dtab] {
       Dtab.read(_parseString(parser, ctx))
     }
 }
+
+class DtabSerializer extends ConfigSerializer[Dtab] {
```

Comment:
If you want to adhere to the letter of the style guide, I might do `object DtabCodec { class Serializer ... class Deserializer }` ; but I don't feel strongly about that.  I _do_ think it's a lot easier for these things to share a home...